www_src/pages/section-2/section-2.jsx

```diff
@@ -170,19 +212,19 @@ var Grid = React.createClass({
     this.gridWidth = undefined;
     this.gridHeight = undefined;
 
+    this.slotWidth = undefined;
```

Comment:
we should probably move these to the `componentWillMount` function instead, instead of redeclaring them on render() every time.